engine/src/main/java/org/destinationsol/game/SolObjectEntityWrapper.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2020 The Terasology Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.game;
+
+import com.badlogic.gdx.math.Vector2;
+import org.destinationsol.entitysystem.EntitySystemManager;
+import org.destinationsol.game.drawables.Drawable;
+import org.terasology.gestalt.entitysystem.entity.EntityRef;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * This is a {@link SolObject} wrapper for an {@link EntityRef}. This is is a patch for the in-between period between
```

Comment:
This is great javadoc 👍 Having this sort of thing for each big important class would be awesome.

Since this is meant to be temporary you might want to throw a TODO into the javadoc or even `@deprecate` it to indicate its doomed nature